src/index.js

```diff
@@ -1,6 +1,37 @@
 import './styles/styles.css';
-import showScoresTable from './modules/showScores';
-import addScoresForm from './modules/addScores';
+import showScoresTable from './showScores';
+import addScoresForm from './addScores';
+import submitScore from './modules/submitScore';
+import getAllScores from './modules/getScores';
 
 showScoresTable();
-addScoresForm();
\ No newline at end of file
+addScoresForm();
+
+const form = document.getElementById('form');
+const userNameInput = document.getElementById('user-name');
+const scoreInput = document.getElementById('user-score');
+const successTxt = document.querySelector('.success');
+const spinner = document.getElementById('spinner');
+const refreshBtn = document.getElementById('refresh-btn');
+
+getAllScores();
+
+form.addEventListener('submit', async (event) => {
+  event.preventDefault();
+  spinner.style.display = 'block';
+  const score = {
+    user: userNameInput.value,
+    score: scoreInput.value,
+  };
+
+  await submitScore(score);
+  spinner.style.display = 'none';
+  successTxt.style.display = 'block';
+  setTimeout(() => {
+    successTxt.style.display = 'none';
+  }, 3000);
+});
+
+refreshBtn.addEventListener('click', () => {
+  getAllScores();
+});
```

Comment:
- [OPTIONAL] Great job far 👏🏻 However there is a small issue with your `Refresh` button implementation related to the UI. Every time I click on the Refresh button, The list of Recent Scores is duplicated on the UI. Kindly prevent this 💯 👍🏻  check the screenshot bellow 👇🏻 

![image](https://user-images.githubusercontent.com/79058364/217741715-00548375-8c8b-47b4-a35f-cab47ffd3289.png)